packages/nvidia_nat_core/src/nat/front_ends/console/console_front_end_plugin.py

```
@@ -105,6 +105,22 @@ def __init__(self, full_config):
+
+        Must be called inside an already-open session context so the auth
+        challenge can reach the user and the token is stored for the
+        workflow run that follows.
+        """
+        for name, cfg in session_manager.config.authentication.items():
+            if cfg.preflight_auth:
+                logger.debug("Preflight auth: authenticating provider '%s'", name)
+                provider = await session_manager.shared_builder.get_auth_provider(name)
+                try:
+                    await provider.authenticate()
+                except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)